src/Fragment/FragmentHandler.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Fragment;
+
+use Contao\CoreBundle\Exception\ResponseException;
+use Contao\CoreBundle\Fragment\Reference\FragmentReference;
+use Contao\PageModel;
+use Psr\Container\ContainerInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Fragment\FragmentHandler as BaseFragmentHandler;
+
+class FragmentHandler extends BaseFragmentHandler
+{
+    /**
+     * @var ContainerInterface
+     */
+    private $renderers;
+
+    /**
+     * @var BaseFragmentHandler
+     */
+    private $fragmentHandler;
+
+    /**
+     * @var FragmentRegistryInterface
+     */
+    protected $fragmentRegistry;
+
+    /**
+     * @var ContainerInterface
+     */
+    private $preHandlers;
+
+    /**
+     * @var array
+     */
+    private $initialized = array();
+
+
+    /**
+     * @param ContainerInterface        $renderers
+     * @param BaseFragmentHandler       $fragmentHandler
+     * @param RequestStack              $requestStack
+     * @param FragmentRegistryInterface $fragmentRegistry
+     * @param ContainerInterface        $preHandlers
+     * @param bool                      $debug
+     */
+    public function __construct(ContainerInterface $renderers, BaseFragmentHandler $fragmentHandler, RequestStack $requestStack, FragmentRegistryInterface $fragmentRegistry, ContainerInterface $preHandlers, bool $debug = false)
```

Comment:
We probably should not call the variable `$renderers`, because `$renderers` is an array in the parent class.